opts/ulimit.go

```diff
@@ -0,0 +1,44 @@
+package opts
+
+import (
+	"fmt"
+
+	"github.com/docker/docker/pkg/ulimit"
+)
+
+type UlimitOpt struct {
+	values *map[string]*ulimit.Ulimit
```

Comment:
And I see where I screwed this up, was pulling in the cli values as &map instead of just map.  This is updated.